apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -231,6 +244,16 @@ export class CaseController {
       update.courtRecordSignatureDate = null
     }
 
+    if (appealState !== undefined) {
+      // TODO: What if user is neither prosecutor nor defender?
+      if (user.role === UserRole.PROSECUTOR) {
+        update.prosecutorPostponedAppealDate = nowFactory()
+      } else if (user.role === UserRole.DEFENDER) {
```

Comment:
I suggest putting the defender appeal transitions into the limitedAccessCase controller.

Reply:
The appeal transition for defenders is now in the limitedAccessController 🙌